tests/unit/test_models.py

```diff
@@ -11,11 +11,12 @@ class TestModels:
     A class to test the models.
     """
 
+    @pytest.mark.parametrize(
+        "model",
+        [pybop.lithium_ion.SPM(), pybop.lithium_ion.SPMe(), pybop.empirical.Thevenin()],
+    )
```

Comment:
I think we should be able to use a fixture here to reduce the duplicate parameterisation blocks below.

```suggestion
    @pytest.fixture(params=[
        pybop.lithium_ion.SPM(),
        pybop.lithium_ion.SPMe(),
        pybop.empirical.Thevenin()
    ])
def model(request):
    return request.param
```